src/components/Choice.java

```diff
@@ -13,7 +13,7 @@
     @Override
     public void parse() {
         String currToken = tokenizer.checkNext();
-        while (!tokenizer.checkToken("text|next scene|conditional|change stat")) {
+        while (tokenizer.checkToken("text|next scene|conditional|change stat")) {
```

Comment:
Removed ! because we want it to keep in loop while tokens match the regex for choice